sdk/iot/Azure.Iot.Hub.Service/api/Azure.Iot.Hub.Service.netstandard2.0.cs

```diff
@@ -43,8 +43,9 @@ public partial class FilesClient
     }
     public enum IfMatchPrecondition
     {
-        UnconditionalIfMatch = 0,
-        IfMatch = 1,
+        Unconditional = 0,
```

Comment:
I'm not sure if this is because you have not rebased but we have only two values here. 
`public enum IfMatchPrecondition
    {
        UnconditionalIfMatch,
        IfMatch
    }`

Reply:
Yeah, this should disappear once Priyanka pulls in the latest from master

Reply:
This is resolved in latest